module/VuFindSearch/src/VuFindSearch/Backend/Summon/PsrConnector.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/**
+ * Summon Search API Interface (Psr implementation)
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
Might as well reflect the current year:

```suggestion
 * Copyright (C) Villanova University 2025.
```